docs/site/Using-strong-error-handler.md

```diff
@@ -0,0 +1,343 @@
+---
+title: 'Using strong-error-handler'
+lang: en
+keywords: LoopBack 4, error handling
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Using-strong-error-handler.html
+summary:
+  Strong-error-handler is an error handler for use in both development and
+  production environments.
+---
+
+In LoopBack 4, the request handling process starts with the app's
+[sequence](https://loopback.io/doc/en/lb4/Sequence.html), a simple class with
+five injected helper methods in the constructor. It is the gatekeeper of all
+requests to the app. Errors are handled by one of the Sequence actions
+[`reject`](Sequence.md#handling-errors), which calls `strong-error-handler`
+package.
+
+# strong-error-handler
+
+This package is an error handler for use in both development (debug) and
+production environments.
+
+## Supported versions
+
+| Current | Long Term Support | End-of-Life |
+| :-----: | :---------------: | :---------: |
+|   4.x   |        3.x        |     2.x     |
+
+Learn more about our LTS plan in
+[docs](http://loopback.io/doc/en/contrib/Long-term-support.html).
+
+## Installation
+
+```bash
+$ npm install --save strong-error-handler
+```
+
+## Use
+
+In production mode, the default implementation of `reject`,
+`strong-error-handler` omits details from error responses to prevent leaking
+sensitive information:
```

Comment:
This section is duplicating existing content in https://loopback.io/doc/en/lb4/Sequence.html#handling-errors. Let's find a way how to avoid that duplication please. For example, the duplicate content on one page can be replaced with a link (a reference) to the other page.